### PR TITLE
#26 - warning for mobile users

### PR DESCRIPTION
**Pull request by samrichca**
https://github.com/overture-stack/dms-jbrowse-components/issues/26

### CODE REVIEW

File: components/pages/explorer/MobileWarning.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import IconButton from '@/components/IconButton';
+import { css, useTheme } from '@emotion/react';
+import { useState } from 'react';
+import { getConfig } from '../../../global/config';
+import { Dismiss as DismissIcon, Error as ErrorIcon } from '../../theme/icons';
+
+const MobileWarning = () => {
+	const theme = useTheme();
+	const [showMobileWarning, setShowMobileWarning] = useState<boolean>(true);
+	const hideMobileWarning = () => setShowMobileWarning(false);
```
Comment (justincorrigible) [inline components/pages/explorer/MobileWarning.tsx:31]: This logic doesn't persist it between sessions. Is this per spec?

File: components/pages/explorer/PageContent.tsx [hunk 2]
```diff
@@ -19,13 +19,16 @@
  *
  */
 
-import { useMemo } from 'react';
 import { css } from '@emotion/react';
 
 import ScrollToTop from '@/components/ScrollToTop';
+
+import { useMemo } from 'react';
```
Comment (justincorrigible) [inline components/pages/explorer/PageContent.tsx:26]: what's the need for this relocation? may I suggest again we use https://www.npmjs.com/package/prettier-plugin-organize-imports in this repo? (thought i had added it in `develop`)

Comment (justincorrigible) [reply components/pages/explorer/PageContent.tsx:26]: scratch that. this is probably the result of that package

File: components/pages/explorer/MobileWarning.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	const { NEXT_PUBLIC_SHOW_MOBILE_WARNING } = getConfig();
+
+	return showMobileWarning && NEXT_PUBLIC_SHOW_MOBILE_WARNING ? (
```
Comment (justincorrigible) [inline components/pages/explorer/MobileWarning.tsx:34]: `NEXT_PUBLIC_SHOW_MOBILE_WARNING` is already checked in PageContent
```suggestion

	return showMobileWarning ? (
```

File: components/pages/explorer/MobileWarning.tsx (hunk 1 above)
Comment (samrichca) [reply components/pages/explorer/MobileWarning.tsx:31]: checked with @patrickdsantos, he wants this to reset on each visit

**Review by justincorrigible — APPROVED** [review]

File: components/pages/explorer/PageContent.tsx [hunk 4]
```diff
@@ -40,13 +44,16 @@ const PageContent = () => {
...
+	const { NEXT_PUBLIC_SHOW_MOBILE_WARNING } = getConfig();
+
 	return useMemo(
 		() => (
 			<div
 				css={css`
 					position: relative;
 				`}
 			>
+        {NEXT_PUBLIC_SHOW_MOBILE_WARNING && <MobileWarning />}
```
Comment (justincorrigible) [inline components/pages/explorer/PageContent.tsx:56]: Looks like the indentation got whacked here?

**Review by justincorrigible — CHANGES_REQUESTED** [review]
Getting build errors in Jenkins, and indentation got weird after a post-approval commit.
image

Looks like conflicts after merging? 🤷

**Review by justincorrigible — APPROVED** [review]
Let's gooo 🚀

**Pull request merged**